README.md

```diff
@@ -89,46 +101,42 @@ Next, very important step: translation from business problem to data science
 ### Business presentation and technical handover
 
 * You will present your results to your "client". Train using language your client can understand and not making it too technical.
```

Comment:
* You will prepare a handover presentation to your client's in-house data scientists. Train writing clear technical communication, which is still business minded. This can be a well documented github repository, jupyter notebook, or a technical presentation. 
* Time...
* These two steps are important to be able to show your portfolio project for future employers (?!)

Reply:
15 minute interview thing 
hiring manager case study
technical team mate case study